lib/web/static/js/app.js

```diff
@@ -0,0 +1,14 @@
+const ResourceList = require('./resource_list.js');
+const listHolders = $('.holder');
+const lists = {}
+
+$.each(listHolders, function(_i, holder) {
+  let list = new ResourceList(holder)
+  let state = holder.dataset.state
+  lists[state] = list
+
+  $(holder).find(".load-more-btn").on("click", function(event) {
```

Comment:
Possible undefined variable: '$' is not defined.

If this is intended to be a global variable, you can supress this warning by adding a  `/*global [VARIABLE NAME] */` comment on this file or set it under the `globals` section of your `.eslintrc` configuration.

[Read more about it here.](http://eslint.org/docs/rules/no-undef)